ios/Rainbow/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>RCTAsyncStorageExcludeFromBackup</key>
+	<false/>
+	<key>CodePushDeploymentKey</key>
+	<string>$(CODE_PUSH_DEPLOYMENT_KEY_IOS)</string>
```

Comment:
how is this related to this PR?

Reply:
I will verify that, I was supposed to add just the first two lines.

Reply:
Fixed, when merging some changes I must have made it duplicate after swaps changes or sth like that